tests/tests_integration/tests_db_connection_integration/test_postgres_integration.py

```diff
@@ -1035,3 +1041,6 @@ def test_postgres_connection_execute_with_legacy_jdbc_options(spark, processing)
 
     options = Postgres.JDBCOptions(query_timeout=30)
     postgres.execute("DROP TABLE IF EXISTS temp_table;", options=options)
+
+    with caplog.at_level(logging.INFO):
+        assert "Detected dialect: 'org.apache.spark.sql.jdbc.PostgresDialect'" in caplog.text
```

Comment:
`with caplog` should wrap `postgres.execute()` call

Reply:
fixed